src/components/Inventory/InventoryCellRenderer.tsx

```diff
@@ -37,7 +37,9 @@ export default function InventoryCellRenderer(props: RendererProps<TableRowType>
   };
 
   const createLinkToInventoryDetail = (iValue: React.ReactNode | unknown[]) => {
-    return <Link to={APP_PATHS.INVENTORY_ITEM.replace(':speciesId', row.species_id.toString())}>{iValue}</Link>;
+    if (!Array.isArray(iValue)) {
```

Comment:
sorry to suggest this so late.. can we make this check inside the Link component instead? That could avoid touching many files

Reply:
To do this we should allow children prop of Link component, to be a ReactNode or unknown[]; and then check that if its an array don't pass it, do we want that? It doesn't seem correct to me that children can be of type unknown[]

Reply:
Adding the isArray check everywhere is not going to be fun and something we will have to do everytime we use Link. None of the use cases are arrays. This is very weird.
Not sure what the real problem is here but handling this in Link would be a better thing to do. Or maybe cast every use case to a ReactNode instead of doing an isarray check.
Or we could do the ReactNode or unknown[] for now with a comment in the code.
I don't know - will leave it to you.

Reply:
we can revisit this later for sure if you want to get this in as-is

Reply:
I casted every use case for now and we can revisit later

Reply:
awesome, thanks @constanzauanini this is much better from a usage standpoint.. looks good 🎉